src/app/shared/pages/collateral/collateral-page/collateral-page.component.html

```diff
@@ -115,6 +116,13 @@ <h2>Sikkerhet i bolig</h2>
           Nei
         </mat-radio-button>
       </mat-radio-group>
+      <mat-error *ngIf="showError()">*Velg om du har medlåntaker</mat-error>
```

Comment:
Trekk "this.isClicked && this.coHabitant == undefined" ut av metoden og rett inn i HTML. Trenger ikke kalle på metode for å bruke if statement her